app/views/browse/service-instance.html

```diff
@@ -0,0 +1,102 @@
+<project-header class="top-header"></project-header>
+<project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+          <alerts alerts="alerts"></alerts>
+          <div ng-if="!loaded" class="mar-top-xl">Loading...</div>
+          <div ng-if="serviceInstance">
+            <h1 class="contains-actions">
+              <div class="pull-right dropdown" ng-hide="!('instances' | canIDoAny)">
+                <button type="button" class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs" data-toggle="dropdown">
+                  Actions
+                  <span class="caret"></span>
+                </button>
+                <a href=""
+                   class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+                   data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+                <ul class="dropdown-menu dropdown-menu-right actions action-button">
+                  <li ng-if="{resource: 'instances', group: 'servicecatalog.k8s.io'} | canI : 'update'">
+                    <a ng-href="{{serviceInstance | editYamlURL}}" role="button">Edit YAML</a>
+                  </li>
+                  <li ng-if="{resource: 'instances', group: 'servicecatalog.k8s.io'} | canI : 'delete'">
+                    <delete-link
```

Comment:
Probably want the same dialog as used here:

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/overview/serviceInstanceRow.js#L84

Let's make it a common component that we can reuse. We'll eventually need to handle bindings.

Reply:
In converting this over to a service, did a comparison on the current directive/component delete link looks like there are obvious "copy" differences, but the core of it is the same. Passing in the additional parameter/option/attribute for "group" and wrapping some of the display logic/ng-ifs with a service instance could solve this while separating out the DOM manipulation from the service

Purpose we either modify the existing, or create a new component, for the delete link for "service instances".

Reply:
@spadgett after going over the modal designs, talked with @ncameronbritt about the style. The style for... 

![screen shot 2017-08-25 at 4 34 31 pm](https://user-images.githubusercontent.com/3761375/29731683-5a1d5292-89b3-11e7-8aef-57d9fa9cc7bc.png)

fits more inline with the design aspect as opposed to the current...

![screen shot 2017-08-25 at 4 34 49 pm](https://user-images.githubusercontent.com/3761375/29731696-6d6a9c42-89b3-11e7-8b1f-299cf305c4d6.png)

Since this seems to fit partially with the "Service Instances Service" I can do a few of things..
- leave the new "Service Instances Service" as is, free floating and convert the existing Provisioned Service "delete" links over to the preferred modal design
- leave the new "Service Instances Service" as is and leave the delete links/buttons alone
- just convert the new "Service Instances Service" by removing the DOM manipulation/modal
- or leave everything as is


@beanh66

Reply:
We probably want something closer to the first dialog, but I'd rather do that in a separate PR. Let's reuse what we have for now and revisit the design for both overview and this page later.